WebRoot/img/animated/ImgLoader.svg

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="utf-8"?><svg width='18px' height='18px' xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" preserveAspectRatio="xMidYMid" class="uil-ring"><rect x="0" y="0" width="100" height="100" fill="none" class="bk"></rect><circle cx="50" cy="50" r="45" stroke-dasharray="183.7831702350029 98.96016858807849" stroke="#999999" fill="none" stroke-width="10"><animateTransform attributeName="transform" type="rotate" values="0 50 50;180 50 50;360 50 50;" keyTimes="0;0.5;1" dur="1s" repeatCount="indefinite" begin="0s"></animateTransform></circle></svg>
```

Comment:
minor, better to format this code as well, if someone wants to understand svg markup